### PR TITLE
feat: Period over Period Big Number comparison chart

### PR DESCRIPTION
**Pull request by eschutho**
### SUMMARY

This is an experimental visualization using the feature flag CHART_PLUGINS_EXPERIMENTAL.
It is a period over period comparison chart for Big Number 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Screenshot 2024-01-30 at 20 34 23 (1)

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by Antonio-RiveroMartnez — APPROVED** [review]

**Review by betodealmeida — APPROVED** [review]
Looks good, but the docs seems to be for a generic plugin, can we fix that?

File: superset-frontend/plugins/plugin-chart-period-over-period-kpi/README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+- software distributed under the License is distributed on an
+- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+- KIND, either express or implied. See the License for the
+- specific language governing permissions and limitations
+- under the License.
+  \*/
+
+# custom-viz
+
+This is the Custom Viz Superset Chart Plugin.
```
[the remark this answers is not in the document]
Comment (eschutho) [reply superset-frontend/plugins/plugin-chart-period-over-period-kpi/README.md:23]: good point 👍

Comment (eschutho) [inline superset-frontend/plugins/plugin-chart-period-over-period-kpi/README.md:23]:
```suggestion
This plugin provides a BigNumber visualization with period over period time comparisons
```

File: superset-frontend/plugins/plugin-chart-period-over-period-kpi/README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+Alternatively, to run the plugin in development mode (=rebuilding whenever changes are made), start the dev server with the following command:
+
+```
+npm run dev
+```
+
+To add the package to Superset, go to the `superset-frontend` subdirectory in your Superset source folder (assuming both the `custom-viz` plugin and `superset` repos are in the same root directory) and run
+
+```
+npm i -S ../../custom-viz
```
Comment (eschutho) [inline superset-frontend/plugins/plugin-chart-period-over-period-kpi/README.md:43]:
```suggestion
npm i -S ../../plugin-chart-period-over-period-kpi
```

File: superset-frontend/plugins/plugin-chart-period-over-period-kpi/README.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+Finally, if you wish to ensure your plugin `tsconfig.json` is aligned with the root Superset project, you may add the following to your `tsconfig.json` file:
+
+```
+"extends": "../../tsconfig.json",
+```
+
+After this edit the `superset-frontend/src/visualizations/presets/MainPreset.js` and make the following changes:
+
+```js
+import { CustomViz } from 'custom-viz';
```
Comment (eschutho) [inline superset-frontend/plugins/plugin-chart-period-over-period-kpi/README.md:74]:
```suggestion
import { PopKPIPlugin } from '@superset-ui/plugin-chart-period-over-period-kpi';
```

File: superset-frontend/plugins/plugin-chart-period-over-period-kpi/README.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+```
+
+to import the plugin and later add the following to the array that's passed to the `plugins` property:
+
+```js
+new CustomViz().configure({ key: 'custom-viz' }),
```
Comment (eschutho) [inline superset-frontend/plugins/plugin-chart-period-over-period-kpi/README.md:80]:
```suggestion
new PopKPIPlugin().configure({ key: 'pop_kpi' }),
```

**Comment by sadpandajoe** [comment]
🏷️ preset:2024.5

**Pull request merged**